tendrl/commons/objects/__init__.py

```diff
@@ -201,7 +202,9 @@ def load(self):
                 pass
 
         _copy = self._copy_vars()
-
+        # Check if self.value already set, use it
+        if self.value.find('{') < 0:
```

Comment:
Including `value` field in `_copy_vars()` causes an error `TypeError: __init__() got an unexpected keyword argument 'value'` while normal `load()` calls for entities, so took this path :) Just changed `_copy_vars` and saw this error again.